mpt_api_client/http/mixins/streaming_mixin.py

```
@@ -307,9 +345,12 @@ async def stream(
                 raise_streaming_error(http_error, path)
             confirm_streaming_mode(response.headers, path)
             async for line in aiter_verified_lines(response, path):
-                model = self._model_class(json.loads(line))  # type: ignore[attr-defined]
+                result = deserialize_stream_record(
+                    json.loads(line),
+                    self._model_class,  # type: ignore[attr-defined]
+                )
                 if progress:
                     await progress.item_processed()  # noqa: WPS476
```

Ruff (0.16.2)

[warning] Invalid rule code in `# noqa`: WPS476

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)